### PR TITLE
Feature/pdct 1385 rename doctype in world map to familycategory or similar

### PR DESCRIPTION
**Pull request by katybaulch**
# What's changed

- Rename docType to famCategory to make it clearer about what the dropdown states represent
- Resize MCF markers
- Inlcude MCF projects in the world map legend

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

### CODE REVIEW

File: src/components/map/WorldMap.tsx [hunk 1]
```diff
@@ -123,14 +123,15 @@ export default function MapChart() {
   const [mapCenter, setMapCenter] = useState<TPoint>([0, 0]);
   const [mapZoom, setMapZoom] = useState(1);
   const [showUnifiedEU, setShowUnifiedEU] = useState(false);
-  const [selectedDocType, setSelectedDocType] = useState<"lawsPolicies" | "unfccc">("lawsPolicies");
+  const [selectedFamCategory, setSelectedFamCategory] = useState<"lawsPolicies" | "unfccc" | "mcf">("lawsPolicies");
```
Comment (jamesgorrie) [inline src/components/map/WorldMap.tsx:126]: For my clarity - these are corpusTypes (with some gnarly MCF hacking in there?

Comment (katybaulch) [reply src/components/map/WorldMap.tsx:126]: these are not corpus types, but family categories - the latter of which we want to remove later down the line in favour of 'genre'.

Comment (katybaulch) [reply src/components/map/WorldMap.tsx:126]: see here: https://github.com/climatepolicyradar/navigator-db-client/blob/500b4f81bb3eeee4111a80e2e33b1e6068eb28fe/db_client/models/dfce/family.py#L19

**Review by PatFawbertMills — APPROVED** [review]
Looks good

**Pull request merged**